docs/en_us/developers/source/accessibility.rst

```diff
@@ -0,0 +1,177 @@
+#######################
+edX Accessibility Guide
+#######################
+
+EdX measures and evaluates accessibility using the World Wide Web Consortium's `Web Content Accessibility Guidelines (WCAG) 2.0 <http://www.w3.org/TR/WCAG/>`_ (Dec. 11, 2008). All features merged into edX are expected to `conform <http://www.w3.org/TR/WCAG20/#conformance>`_ to `Level AA <http://www.w3.org/TR/WCAG20/#cc1>`_ of this specification. This guide is intended to extend the guidance available in WCAG 2.0, with a focus on features frequently found in the Open edX platform as well as Learning and Content Management Systems in general.
+
+Introduction
+************
+
+The core mission of edX is to expand access to education for everyone. We expect that any user interfaces that are developed for the Open edX platform should be usable by everyone, regardless of any physical limitation they may have. The Open edX platform is used every day by people who might not be able to see, hear or might not have the dexterity to use traditional modes of computer interaction (such as the mouse). Understanding a few core concepts about how people with disabilities use the web and web applications should make the guidance in this document a little easier to understand.
+
+* A person without vision cannot use a mouse and neither can those with neurological conditions that prevent the kind of precise hand-eye coordination required to use a mouse.
+* Many people with disabilities use "`Assistive Technology <http://www.w3.org/TR/WCAG20/#atdef>`_" (such as screen readers) to interact with their computers, web browsers, and web applications.
+* Anything conveyed visually (such as an element's role, state or other properties) must also be conveyed programmatically so that assistive technologies know the element's role, what state it's currently in and how users can interact with it. *Protip:* This information often comes for free when you use native HTML5 elements.
+* Many people with mobility impairments use custom keyboards or keyboard emulators.
+* Some people rely on speech to interact with web applications and need to be able to address interactive elements.
+* Someone who cannot hear or hear well requires visible text captions for any audible content or an equivalent visual cue.
+* Users may customize their operating system, browser or web page display properties to make web applications easier for them to use (like reversing contrast, increasing font size, or removing images). 
```

Comment:
@catong , it seems to me that this overview (complete file up to this point) would be useful for course authors as well. Perhaps in the future we could break this into several files and make some of them shared so that the same information is available to multiple audiences.